### PR TITLE
doc(gcb): update the markdown generators to use gcb files

### PR DESCRIPTION
**Pull request by devjgm**
Fixes: #6176

The markdown generators now use the Dockerfiles in
`ci/cloudbuild/dockerfiles` instead of `ci/kokoro/install`.

---
This change is [Reviewable](https://reviewable.io/reviews/googleapis/google-cloud-cpp/6385)

### CODE REVIEW

File: ci/generate-markdown/generate-packaging.sh [hunk 1]
```diff
@@ -225,67 +225,40 @@ the case, the instructions describe how you can manually download and install
...
+    -e '/^.*\[END packaging.md\].*$/,$d' "$1"
+}
+
+# A "map" (comma separated) of dockerfile -> summary.
+DOCKER_DISTROS=(
+  "demo-fedora.Dockerfile,Fedora (33)"
+  "demo-opensuse-leap.Dockerfile,openSUSE (Leap)"
+  "demo-ubuntu-focal.Dockerfile,Ubuntu (20.04 LTS - Focal Fossa)"
+  "demo-ubuntu-bionic.Dockerfile,Ubuntu (18.04 LTS - Xenial Xerus)"
+  "demo-ubuntu-xenial.Dockerfile,Ubuntu (16.04 LTS - Xenial Xerus)"
```
Comment (coryan) [inline ci/generate-markdown/generate-packaging.sh:240]: 16.04 is Xenial Xerus, 18.04 is Bionic Beaver:

https://wiki.ubuntu.com/Releases

File: doc/packaging.md [hunk 2]
```diff
@@ -587,19 +589,20 @@ sudo ldconfig
...
+cmake -DBUILD_TESTING=OFF -DCMAKE_INSTALL_PREFIX="${PREFIX}" -H. -Bcmake-out
+cmake --build cmake-out -- -j "$(nproc)"
+cmake --build cmake-out --target install
 ```
 
 </details>
 
 <details>
-<summary>Ubuntu (18.04 LTS - Bionic Beaver)</summary>
+<summary>Ubuntu (18.04 LTS - Xenial Xerus)</summary>
```
Comment (coryan) [inline doc/packaging.md:605]: This is a side effect of that previous nit.

**Review by coryan — APPROVED** [review]
A small nit in naming of Ubuntu releases, the rest LGTM.

File: ci/generate-markdown/generate-packaging.sh (hunk 1 above)
Comment (devjgm) [reply ci/generate-markdown/generate-packaging.sh:240]: Oops! Fixed.

**Pull request merged**